### PR TITLE
Mac Video Game Controllers

### PR DESCRIPTION
**Pull request by ThoughtbotThom**
added til for pairing windows xbox controllers with OS X

### CODE REVIEW

File: OSX/macController.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Mac Controller Support
+
+Attempting to play video games natively on OS X has always been a rough proposition due to lack of supported games and hardware; but for years Colin Munro allowed us to utilize a Windows Xbox360 controller thru a [custom driver](http://tattiebogle.net/index.php/ProjectRoot/Xbox360Controller/OsxDriver). 
+
+Sadly, Yosemite kext signing requirements broke Colin's last build; though fortunately the project has been [forked to GitHub](https://github.com/d235j/360Controller), and has since reimplemented full controller support, rumble, LED, & System Preferences prefpane functionality. Game on!
```
Comment (JoelQ) [inline OSX/macController.md]: kext ?

Comment (ThoughtbotThom) [inline OSX/macController.md]: kernel extension

File: OSX/macController.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#Mac Controller Support
+
+Attempting to play video games natively on OS X has always been a rough proposition due to lack of supported games and hardware; but for years Colin Munro allowed us to utilize a Windows Xbox360 controller thru a [custom driver](http://tattiebogle.net/index.php/ProjectRoot/Xbox360Controller/OsxDriver).
```
Comment (JoelQ) [inline OSX/macController.md]: thru -> through

File: OSX/macController.md (hunk 1 above)
Comment (JoelQ) [inline OSX/macController.md]: Thoughts on not abbreviating here?

**Pull request merged**